daml/ClaimToken/Service.daml

```diff
@@ -0,0 +1,65 @@
+module ClaimToken.Service where
+
+import ClaimToken.Model
+import DA.Finance.Asset
+
+template Service with
+  operator: Party
+  provider: Party
+  owner: Party
+
+    where
+      signatory operator, provider
+      observer owner
+
+      key (operator, provider, owner) : (Party, Party, Party)
+      maintainer key._1
+    
+      controller owner can 
+        nonconsuming Mint : ContractId AssetDeposit
+          with 
+            assetDepositCid : ContractId AssetDeposit
+            claimTokenR : ClaimTokenR 
```

Comment:
This probably needs to be `ContractId ClaimTokenR` so that you cannot just pass any data in here.

Reply:
@GeorgSchneider this have similar issue as the HolidayCalendar that we pass in for reporting service.
ClaimToken and ClaimTokenR should be visible to public (and every party should be able to fetch them), but when the owner of the claim token are fetching it, we are not able to do something like
`controller public, owner can` (and we don't know who will own certain bond when we originate them, so we can't really put owners as observer to ClaimToken or ClaimTokenR contracts

Reply:
as discussed with @GeorgSchneider, we can use the flexible choice to return id of ClaimToken and ClaimTokenR and make those visible by public to resolve the issue here.
Updated as suggested.